test/static/test_licenses.py

```diff
@@ -0,0 +1,24 @@
+######################################################################
+#
+# File: test/static/test_licenses.py
+#
+# Copyright 2020 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from glob import glob
+from itertools import islice
+
+import pytest
+
+
+def test_files_headers():
```

Comment:
it does not check the date

Reply:
Nither previous CI and pre-commit hook. In Travis there was a logic for that, but commented out, in pre-commit hook there wasn't.

Do we really want to break the CI every 1st of January?